sdks/java/core/src/main/java/org/apache/beam/sdk/options/PipelineOptionsFactory.java

```diff
@@ -581,6 +582,7 @@ private static void initializeRegistry() {
   /**
    * Validates that the interface conforms to the following:
    * <ul>
+   *   <li>Every interface must be inherited from {@link PipelineOptions}.
```

Comment:
`Every interface must be inherited from {@link PipelineOptions}.` -> `Every inherited interface of {@code iface} must extend PipelineOptions except for PipelineOptions itself.`